packages/builder/sample-project/algob.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  networks: {
+    localhost: {
```

Comment:
No `default`?

Reply:
No need for `default`  -- if user is not providing it, it will come from the default config (this is also an original behaviour)

Reply:
The default config is hardcoded in a runtime.